bin/dump_wiki

```diff
@@ -42,7 +44,11 @@ if (argv.h) {
     argParser.showHelp();
     process.exit(1);
 }
-
+if (!argv.dataBase && !argv.saveDir) {
```

Comment:
@apergos, in RESTBase. The htmldumper script only spiders the contents in that case, and throws away the result.